app/scripts/TrackRenderer.jsx

```diff
@@ -237,13 +524,19 @@ class TrackRenderer extends React.Component {
   }
 
   componentDidMount() {
+    if (!this.element) {
+      throw new Error('Component did not mount, this.element is not defined.');
+    }
     this.elementPos = this.element.getBoundingClientRect();
     this.elementSelection = select(this.element);
-    this.svgTrackAreaSelection = select(this.svgTrackArea);
```

Comment:
`this.svgTrackArea` does not exist in this file and there are no references across the code base. Same with `svgTrackAreaSelection`.

@pkerpedjiev